Script/PRONTO.py

```diff
@@ -420,13 +540,21 @@ def update_ppt_template_data(inpred_node,ipd_no,ipd_gender,ipd_age,ipd_diagnosis
 		tf6.paragraphs[0].text = tumor_content
 		tf6.paragraphs[0].font.size = Pt(14)
 		tf6.paragraphs[0].alignment = PP_ALIGN.CENTER
```

Comment:
All the textbox6 related commands could go to a new function called something like `set_layout_textbox_tumor_content`. Similarly, commands related to other textboxes could be split to smaller functions with meaningful naming.

Reply:
Referenced to an issue now.